ui/src/components/inputs/MonacoEditor.vue

```diff
@@ -147,21 +147,29 @@
                         modified: modifiedModel
                     });
                 } else {
-                    if (this.schemas !== undefined) {
-                        setDiagnosticsOptions({
-                            enableSchemaRequest: true,
-                            hover: true,
-                            completion: true,
-                            validate: true,
-                            format: true,
-                            schemas: this.schemas.map(r => {
-                                return {
-                                    uri: r,
-                                    fileMatch: ["*"]
-                                }
-                            })
-                        });
-                    }
+                    setDiagnosticsOptions({
```

Comment:
this function is global so must be at top of the page and not hidden inside the vue object